runner/src/main/twirl/org/globalnames/parser/runner/web/views/api.scala.html

```diff
@@ -0,0 +1,36 @@
+@import org.globalnames.parser.runner.web.views.MenuItem
+
+@(menu: Seq[MenuItem])
+
+@layout(title = "GNParser API", menu) {
+  <section class="parser api">
+    <div class="grid">
+      <div class="unit whole">
+        <h2 id="api">Application Programming Interface (API)</h2>
+
+        <p>Web-based parser service includes a RESTful interface to parsing
+        functionalilty.  Both GET and POST methods are supported.</p>
+
+        <h3 id="get">GET</h3>
+
+        <p>
+          Append a vertical line separated array of strings to your domain url.
+          Make sure that '&amp;' in the names are escaped as '%26',
+          and spaces are escaped as '+'.
+        </p>
+
+        <p>
+          <code>/api?q=Aus+bus|Aus+bus+D.+%26+M.,+1700</code>
```

Comment:
Can you change a year to > 1758 ? Parser actually should not parse years less than 1750